### DISCUSSION TITLE
Filter toolbox with LINQ search

### DISCUSSION THREAD

**Discussion by pawarbi** [TabularEditor/TabularEditor3#1026] [Ideas]
@otykier where do I find Filter toolbox in TE3 to use LINQ, similar to TE2? I can't seem to find it anywhere. I don't see it in customize toolbox. The search feature in TOM Explorer doesn't recognize LINQ expressions. Do I need to enable any options ? Thanks.

https://docs.tabulareditor.com/te2/Advanced-Filtering-of-the-Explorer-Tree.html

**Comment by otykier** [comment]
It's not currently possible to use LINQ expressions in the TOM Explorer filter in TE3. However, if you press Ctrl+F (Find), you can use LINQ expressions when searching across the model.

LINQ expressions in the TOM Explorer filter is a planned feature.

**Comment by zenzeinet** [comment]
Just a quick thumbs up to have this implemented, when you need it you really miss it, coming from TE2 :)

**Reply by zenzeinet** [reply]
Another thumbs-up, this seems forgotten!

**Reply by mlonsk** [reply]
Hi @zenzeinet 
It is not forgotten, we have not had the resources to work on it yet. 
I appreciate its been a wish for a good while now, and you raising it again is a good indication that this is still an important feature for you. I will discuss with the developer team how this can fit into our priorities in the coming months. 
Thank you for your patience. 

Edit: Seems to be the same as #1045
